src/PHPUnit/CrudTestBaseAbstract.php

```diff
@@ -563,7 +565,7 @@ protected function getToken ():string {
 
         $testUser = $userRepo->findOneBy(['id'=>1]);
 
-        $response = $this->json('POST', '/auth/authenticate', ['email' => $testUser->getEmail(), 'password' => $testUser->getPassword()]);
+        $response = $this->json('POST', '/auth/authenticate', ['email' => $testUser->getEmail(), 'password' => 'password']);
```

Comment:
Isn't the password something we are setting in the env file? I think this should reference the env files: BASE_USER_PASSWORD